src/main/resources/i18n/Enums_en.properties

```diff
@@ -19,18 +19,6 @@ accelerator.DeliverableCategory.SupplementalFiles=Supplemental Files
 accelerator.DeliverableCategory.VerraNonPermanenceRiskToolNPRT=Verra Non-Permanence Risk Tool (NPRT)
 # Noun, as in a legal document
 accelerator.DeliverableType.Document=Document
-accelerator.ScoreCategory.Carbon=Carbon
-accelerator.ScoreCategory.Finance=Finance
-accelerator.ScoreCategory.Forestry=Forestry
-accelerator.ScoreCategory.Legal=Legal
-accelerator.ScoreCategory.Community=Community
-accelerator.ScoreCategory.GIS=GIS
-accelerator.ScoreCategory.ClimateImpact=Climate Impact
-accelerator.ScoreCategory.ExpansionPotential=Expansion Potential
-accelerator.ScoreCategory.ExperienceAndUnderstanding=Experience and Understanding
-accelerator.ScoreCategory.OperationalCapacity=Operational Capacity
-accelerator.ScoreCategory.ResponsivenessAndAttentionToDetail=Responsiveness and Attention to Detail
-accelerator.ScoreCategory.ValuesAlignment=Values Alignment
```

Comment:
Phrase will re-add these next time it exports strings as a PR. Removing them here is fine, it'll just also require someone with Phrase admin rights (I'm one of them) to go delete these strings from our strings list in the Phrase UI once this is merged. Their GitHub import feature can only add and update strings, not delete them.

This is just an FYI so you're not puzzled when this deletion doesn't stick.

Reply:
I've deleted these from Phrase.